packages/terra-responsive-element/docs/README.md

```diff
@@ -6,6 +6,8 @@ The Responsive Element can be set to be responsive to the parent of the componen
 
 ## Additional Notes
 
+The ResponsiveElement observes the total width. Meaning padding, borders except the margins set on the parent are included in the calculation. Modifying box-sizing will impact how these properties are used in an elements width calculation. For more details see: https://css-tricks.com/almanac/properties/b/box-sizing/
```

Comment:
We might clean this sentence up, it is a little confusing as it is. Maybe something along the lines of:

The ResponsiveElement observes the total width of the bound container. Padding and borders are included in the calculation to determine the available width. Margins are not included. For breakpoint accuracy it is not recommended to set padding on the parent container when responsiveTo is set to parent.